database/seeders/DatabaseSeeder.php

```diff
@@ -11,8 +11,8 @@ class DatabaseSeeder extends Seeder
     /**
      * Seed the application's database.
      */
-    public function run(): void
-    {
+    public function run(): void    {
```

Comment:
他のSeederファイルの記述が統一されるようにお願いします。
```suggestion
    public function run(): void
    {
```